docs/glossary.rst

```diff
@@ -38,46 +24,57 @@ Astropy Glossary
        For example, ``distance : quantity-like ['length']``
 
    angle-like
-      :term:`quantity-like`, but interpreted by an angular
-      `~astropy.units.SpecificTypeQuantity`, like `~astropy.coordinates.Angle`
-      or `~astropy.coordinates.Longitude` or `~astropy.coordinates.Latitude`.
-      Note that the interpretation of units in strings depends on the class --
-      ``Quantity("180d")`` is 180 days, while ``Angle("180d")`` is 180 degrees
-      -- so make sure the string parses as intended for ``Angle``.
+      :term:`quantity-like` and a valid initializer for `~astropy.coordinates.Angle`.
+      The ``unit`` must be an angle equivalent to radians. A string input is interpreted
```

Comment:
Maybe just "The ``unit`` must be an angular unit." or perhaps just "The ``unit`` must be angular."

Reply:
Fixed.